next/pages/dashboard/webforms/[webformName]/[webformSubmissionUuid].tsx

```diff
@@ -69,14 +70,18 @@ export const getServerSideProps: GetServerSideProps<
     `/${locale}/webform_rest/${params.webformName}/complete_submission/${params.webformSubmissionUuid}`,
   );
 
-  const result = await drupal.fetch(url.toString(), {
-    method: "GET",
-    headers: {
-      "Content-Type": "application/json",
-      // Pass the token to authenticate the request:
-      Authorization: `Bearer ${session.accessToken}`, // eslint-disable-line @typescript-eslint/no-base-to-string
-    },
-  });
+  const result = await pRetry(
+    () =>
+      drupal.fetch(url.toString(), {
+        method: "GET",
+        headers: {
+          "Content-Type": "application/json",
+          // Pass the token to authenticate the request:
+          Authorization: `Bearer ${session.accessToken}`, // eslint-disable-line @typescript-eslint/no-base-to-string
+        },
+      }),
+    { retries: 5 },
+  );
```

Comment:
Testing locally, a "see more" submission link from my dashboard page (e.g. `https://frontend.lndo.site/dashboard/webforms/contact/86c2b13c-9aa6-4abe-bd00-197787fe46e1`) gives a 404 not found? Not sure if related to this new feature or not as the code LGTM here and the dashboard page itself works fine.

Reply:
ah! it seems like it's an issue in the backend

I will investigate more, this is the matching issue in drupal.org: https://www.drupal.org/project/webform_rest/issues/3415211

Reply:
ok, fixed by adding a patch in 5bf3e689c340dcfed8732ca9c5dc54d25f95ecc9